to.us.scrollprotocol.Profectus.yml

```diff
@@ -0,0 +1,66 @@
+id: to.us.scrollprotocol.Profectus
+runtime: org.freedesktop.Platform
+runtime-version: "23.08"
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.golang
+command: profectus
+finish-args:
+  # X11 + XShm access
+  - --share=ipc
+  - --socket=fallback-x11
+  # Wayland access
+  - --socket=wayland
+  # GPU acceleration if needed
+  - --device=dri
+  # Needs to talk to the network:
+  - --share=network
+  # Needs to save files locally
+  - --filesystem=xdg-download:rw
+  # Input
+  - --device=input
+  # Audio
+  - --socket=pulseaudio
+  - --filesystem=xdg-run/pipewire-0:ro
+build-options:
+  env:
+    - GOBIN=/app/bin
+    - GOROOT=/usr/lib/sdk/golang
+modules:
+  - name: sdl2
+    buildsystem: autotools
+    sources:
+      - type: archive
+        url: https://github.com/libsdl-org/SDL/releases/download/release-2.30.3/SDL2-2.30.3.tar.gz
+        sha256: 820440072f8f5b50188c1dae104f2ad25984de268785be40c41a099a510f0aec
+  - name: sdl2-image
+    buildsystem: autotools
+    sources:
+      - type: archive
+        url: https://github.com/libsdl-org/SDL_image/releases/download/release-2.8.2/SDL2_image-2.8.2.tar.gz
+        sha256: 8f486bbfbcf8464dd58c9e5d93394ab0255ce68b51c5a966a918244820a76ddc
+  - name: sdl2-ttf
+    buildsystem: autotools
+    sources:
+      - type: archive
+        url: https://github.com/libsdl-org/SDL_ttf/releases/download/release-2.22.0/SDL2_ttf-2.22.0.tar.gz
+        sha256: d48cbd1ce475b9e178206bf3b72d56b66d84d44f64ac05803328396234d67723
+  - name: sdl2-gfx
+    buildsystem: autotools
+    sources:
+      - type: archive
+        url: https://sourceforge.net/projects/sdl2gfx/files/SDL2_gfx-1.0.4.tar.gz/download
+        sha256: 63e0e01addedc9df2f85b93a248f06e8a04affa014a835c2ea34bfe34e576262
+        archive-type: tar-gzip
+  - name: profectus
+    buildsystem: simple
+    build-commands:
+      - $GOROOT/bin/go build -trimpath -o profectus
+      - install -Dm00755 profectus $FLATPAK_DEST/bin/profectus
+      - install -Dm00644 Icon.png $FLATPAK_DEST/share/icons/hicolor/256x256/apps/$FLATPAK_ID.png
+      - install -Dm00644 $FLATPAK_ID.desktop $FLATPAK_DEST/share/applications/$FLATPAK_ID.desktop
+      - install -Dm00644 $FLATPAK_ID.metainfo.xml $FLATPAK_DEST/share/appdata/$FLATPAK_ID.metainfo.xml
```

Comment:
in this file: `  <developer id="tld.vendor">` is invalid. the id is an example.

Reply:
The id should be a reverse domainn name or something?

Reply:
https://www.freedesktop.org/software/appstream/docs/chap-Metadata.html#tag-developer

Reply:
Does a developer name or username get added to the end of the id? Or is it just `to.us.scrollprotocol` that should be used.

Reply:
id="to.us.scrollprotocol"

Reply:
@bbhtt I already went ahead and used `to.us.scrollprotocol.clseibold`, but I can change it to just the domain if it should be that I guess.